lib/javascripts/base_app.js

```diff
@@ -55,17 +48,26 @@ function BaseApp(zafClient, data) {
   this.I18n = { t: I18n.t };
   registerHelpers(this);
   bindEvents(this);
-  let evt = { firstLoad: true };
+
   this._metadata = data.metadata;
   this._context = data.context;
-  zafClient.get('currentUser.locale').then(function(data) {
-    I18n.loadTranslations(data['currentUser.locale']);
-    if (this.defaultState) {
-      this.switchTo(this.defaultState);
-    }
-    resolveHandler(this, 'app.created')();
-    resolveHandler(this, 'app.activated')(evt, evt);
-  }.bind(this));
+  this._storage = new Storage(this._metadata.installationId);
+
+  let view = new View({ onRender: () => {
```

Comment:
You'd typically mixin by a class-level operation, no? Like, create a `AutoResizeView` by mixing in the `onRender` method, then instantiate the new class/object.  In your case, you're modifying the instance each time, right?